source/background.ts

```diff
@@ -136,10 +141,14 @@ async function pluginNeeds(): Promise<{
 		pluginNames.includes(plugin.name),
 	);
 	const needPageContent = Boolean(
-		selectedPlugins.findIndex((plugin) => plugin.requires.document) >= 0,
+		selectedPlugins.findIndex((plugin) =>
+			plugin.requires.has(Requirements.Document),
+		) >= 0,
```

Comment:
Mulig forbedring, du bestemmer selv om du vil lave den:
Nu hvor vi bruger sets, kan vi faktisk bare lave et loop hvor vi kører [Set.union()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Set/union) på alle plugins, og så til sidst kan du lave de to tjek uden det der lidt hacky `findIndex`.